conjur/cli.py

```diff
@@ -20,6 +19,9 @@
 # Internals
 from conjur.argparse_wrapper import ArgparseWrapper
 from conjur.client import Client
+from conjur.init_logic import InitLogic
+from conjur.init_controller import InitController
+from conjur.input_conjurrc import InputConjurrc
```

Comment:
`input_conjurrc` needs a better name. "input" here is highly ambiguous. This may also need to be moved to `conjur/init/` module

Reply:
I intended for `input_conjurrc` to hold all user input data that would be given to us from the user to configure the conjurrc. Would `conjurrc_data` be better?

Reply:
why not just name it `conjurrc`?

Reply:
because it holds the data. I think conjurrc_data is fine